rand_distr/src/lib.rs

```diff
@@ -94,12 +103,15 @@ pub use self::unit_ball::UnitBall;
 pub use self::unit_circle::UnitCircle;
 pub use self::unit_disc::UnitDisc;
 pub use self::unit_sphere::UnitSphere;
-pub use self::utils::Float;
+pub use num_traits::Float;
```

Comment:
Maybe replace it with `pub use num_traits;`? Note that some distributions also use the `FloatConst` trait.

Reply:
Either way it's a breaking change. Yes, probably better to re-export `num_traits` (in part because it makes it clear that there is a breaking change).